packages/ol/no-strip.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/GNUmakefile b/GNUmakefile
+index 160b044f..fcbc2cd1 100644
+--- a/GNUmakefile
++++ b/GNUmakefile
+@@ -257,7 +257,7 @@ libol.so:
+ # real name of
+ olvm: vm
+ 	cp $< $@
+-	strip $@
++	#strip $@
```

Comment:
You can replace `strip` with `llvm-strip` which is already in PATH.

Reply:
Thanks. Replaced.